mautic/templates/deployment.yaml

```diff
@@ -50,20 +48,16 @@ spec:
             tcpSocket:
               port: http
           volumeMounts:
-            - name: mautic-data
-              mountPath: /var/www/html
+           {{- toYaml .Values.volumeMounts | nindent 12 }}            
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
+            
       volumes:
         - name: mautic-data
-          {{- if .Values.persistence.enabled }}
```

Comment:
Why you removed persistence configuration? =\

Reply:
We don't need to use "if" there because we are not going to use "emptyDir: {}" volume

Reply:
Why we not going to use "emptyDir: {}" ? we use it incase of persistence disabled.

Reply:
Цe cannot use "emptyDir: {}" because it deletes the contents of the folders it is connected to every time the pod is recreated, it's even worse than not having any volume at all

Reply:
For not persistence storage we are interested in one thing, when it is up mautic is working.